src/class/methods.rs

```diff
@@ -209,3 +200,25 @@ pub trait PyMethodsInventory: inventory::Collect {
 pub trait HasMethodsInventory {
     type Methods: PyMethodsInventory;
 }
+
+#[derive(Debug)]
+pub struct NulByteInString(pub(crate) &'static str);
```

Comment:
Do we really want to expose this as a public error type?
I think `std::ffi::NulError` is better at least for `as_method_def`, though then we need a bit more glue codes to display specific error messages.

Reply:
Actually I think that `as_method_def` can now be `pub (crate)` - so I'll apply that and also to this error type.

Reply:
Sounds good